### PR TITLE
Make searchbox visible in https://www.jenkins.io/ when the collapsed content is already shown on small-screen

### PR DESCRIPTION
**Pull request by lakshmishreea122003**
I came up with this PR in my attempt to fix the issue https://github.com/jenkins-infra/jenkins.io/issues/6046 .

### CODE REVIEW

File: src/jio-navbar.ts [hunk 1]
```diff
@@ -197,6 +197,9 @@ export class Navbar extends LitElement {
     let searchboxHtml = null;
     if (this.showSearchBox) {
       searchboxHtml = html`<jio-searchbox></jio-searchbox>`;
+      if (this.menuToggled) {
+        this._clickCollapseButton;
```
Comment (halkeye) [inline src/jio-navbar.ts:201]: I'm not sure what the intention here is.

1) _clickCollapseButton is a function, so it should be this._clickCollapseButton()
2) This is the render function, so every time the component renders at all, it'll toggle the menu 
3) showSearchBox is a attribute to decide if the searchbox should be shown on the menu bar, its not an action as you sorta think it is.

You may be able to get at what you are trying to do by adding an onclick to jio-searchbox, or somehow add an event handler to the algolia stuff.

Comment (lakshmishreea122003) [reply src/jio-navbar.ts:201]: I had misunderstood certain aspects. I shall add onclick to jio-searchbox as you have advised. Thanks for the explanation.

**Comment by lakshmishreea122003** [comment]
@halkeye I did add onclick to jio-searchbox as you had advised. Please do let me know if my approach was correct. If not I shall make the required changes

File: src/jio-navbar.ts [hunk 2]
```diff
@@ -292,6 +292,12 @@ export class Navbar extends LitElement {
     this.menuToggled = !this.menuToggled;
   }
 
+  private _handleSearchboxClick() {
+    if (this.menuToggled) {
+      this._clickCollapseButton();
```
Comment (lakshmishreea122003) [inline src/jio-navbar.ts:297]: if it is not appropriate to use this._clickCollapseButton(); then I shall replace it with this.menuToggled = false; . If both these approaches are wrong then please do advise as to how I could approach to fix the issue https://github.com/jenkins-infra/jenkins.io/issues/6046

**Comment by halkeye** [comment]
image

https://deploy-preview-62--jenkins-io-components.netlify.app/iframe.html?args=&id=example-navbar--searchbox-enabled&viewMode=story

Are you testing this out at all? or just blindly submitting PRs? Are you able to run storybook (`npm run dev`)? Wondering if i need to improve docs at all.

**Comment by lakshmishreea122003** [comment]
Now I have tested it on gitpod and it works as desired . 
image

**Comment by halkeye** [comment]
I think this is good, I just want to poke around storybook before merging, but seems good at first glance.

**Review by iamrajiv — APPROVED** [review]

**Comment by lakshmishreea122003** [comment]
Are there any more improvements to be made in order for this PR to be merged? Kindly let me know

**Pull request merged**